database.js

```diff
@@ -0,0 +1,31 @@
+const bcrypt = require('bcrypt');
```

Comment:
your file is called 'database.js' but inside you are using logic for user. Thats bad. Split logic and files with correct naming